code/game/objects/items/mountable_frames/newscaster_frame.dm

```diff
@@ -1,19 +1,47 @@
-/obj/item/mounted/frame/newscaster_frame
+/obj/item/mounted/frame/display
+	icon = 'icons/obj/status_display.dmi'
+	icon_state = "frame"
+	item_state = "syringe_kit"
+	materials = list(MAT_METAL=6000, MAT_GLASS=2000)
+	mount_requirements = MOUNTED_FRAME_SIMFLOOR | MOUNTED_FRAME_NOSPACE
+	metal_sheets_refunded = 3
+	glass_sheets_refunded = 1
+	var/build_path
+
+/obj/item/mounted/frame/display/do_build(turf/on_wall, mob/user)
+	var/obj/machinery/status_display/N = new build_path(get_turf(src), get_dir(on_wall, user), 1)
```

Comment:
what is the `1` here being used for?

Reply:
Did search and test, think its only for the amount, air alarm and apc frame both have it too.